Formula/brogue.rb

```diff
@@ -20,30 +16,27 @@ class Brogue < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "d972aa4e6b4d0a1d80e28b904bba68d2fc938776edf9094a3d22c1af56644095"
   end
 
+  depends_on "sdl2"
+  depends_on "sdl2_image"
+
   uses_from_macos "ncurses"
 
   # put the highscores file in HOMEBREW_PREFIX/var/brogue/ instead of a
   # version-dependent location.
-  patch do
-    url "https://raw.githubusercontent.com/Homebrew/formula-patches/c999df7dff/brogue/1.7.4.patch"
-    sha256 "ac5f86930a0190146ca35856266e8e8af06ac925bc8ae4c73c202352f258669c"
-  end
+  patch :DATA
```

Comment:
This should be upstreamed now (although probably not with our hardcoded paths)

Reply:
Based on alternative script https://github.com/tmewett/BrogueCE/blob/v1.11.1/linux/brogue-multiuser.sh, something similar should be possible by modifying wrapper `brogue`. The above script uses user-specific directory via:
```sh
broguedir="/opt/brogue"  # where you set DATADIR to (copy the contents of 'bin' to here)
userdir="${XDG_DATA_HOME:-$HOME/.local/share}/Brogue"  # where you want user files to go

mkdir -p "$userdir"
cd "$userdir"
exec "$broguedir"/brogue "$@"
```

We can just do something like:
```sh
#!/bin/bash
cd "#{var}/brogue"
exec "#{libexec}/brogue" "$@"
```

Could run `mkdir` in script, or move `mkdir` into `post_install` as I don't think the empty directory from install block is retained in bottle.

Reply:
I'm fine with either solution though I think configuring the location at build time is a little nicer.